code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1270,7 +1270,7 @@ var/global/list/damage_icon_parts = list()
 		var/icon/tailw_s = new/icon("icon" = 'icons/effects/species.dmi', "icon_state" = "[tail]w_s")
 		if(species.bodyflags & HAS_SKIN_COLOR)
 			tailw_s.Blend(rgb(r_skin, g_skin, b_skin), ICON_ADD)
-		if(tail_marking_icon)
+		if(tail_marking_icon && !tail_marking_style.tails_allowed)

```

Comment:
Won't this prevent any and all tail markings specific to a certain tail from rendering?